### ISSUE TITLE
add WCAG compliance to the cards/blades containing links

### ISSUE THREAD

**Issue by ramboz** [hlxsites/delta#121]
Cards and blades with a navigation link and that get the red ">" arrow are not focusable via tab as the link itself is hidden in the markup. So we need to make those focusable and support activating the link on space/enter to trigger the navigation.

We have a few options:

1. rewrap the card into an `<a>` so we get the behavior for free from the browser
2. explicitly set the tabindex on the card to `tabindex="0"`, and handle space/enter keydown to trigger the hidden link
